reference/5.1/Microsoft.PowerShell.Core/About/about_Job_Details.md

```diff
@@ -1,97 +1,96 @@
 ---
-description:  Provides details about background jobs on local and remote computers. 
+description: Provides details about background jobs on local and remote computers.
 keywords: powershell,cmdlet
 Locale: en-US
-ms.date: 11/28/2017
+ms.date: 10/16/2020
 online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_job_details?view=powershell-5.1&WT.mc_id=ps-gethelp
 schema: 2.0.0
 title: about_Job_Details
 ---
-
 # About Job Details
 
-## SHORT DESCRIPTION
-
+## Short description
 Provides details about background jobs on local and remote computers.
 
-## DETAILED DESCRIPTION
+## Detailed description
 
 This topic explains the concept of a background job and provides technical
 information about how background jobs work in PowerShell.
 
-This topic is a supplement to the [about_Jobs](about_Jobs.md) and
-[about_Remote_Jobs](about_Remote_Jobs.md) topics.
+This topic is a supplement to the [about_Jobs](about_Jobs.md),
+[about_Thread_Jobs](/powershell/module/microsoft.powershell.core/about/about_Thread_Jobs.md),
```

Comment:
ThreadJob module does not ship in 5.1 so we don't have docs for it in that version. However, it can be installed in 5.1, so we want to link to the docs we have.